package.json

```diff
@@ -37,7 +37,7 @@
     "graphql": "^16.6.0",
     "graphql-request": "^5.0.0",
     "lodash": "^4.17.21",
-    "next": "13.1.6",
+    "next": "13.5.7",
```

Comment:
This is the LTS version of Next 13, no breaking changes coming from our older version